src/components/admin/admin.service.ts

```diff
@@ -1,151 +1,141 @@
-
-    const session = this.db.driver.session();
-    const wait = [];
-
-    // ORGANIZATION
-    wait.push(
-      session.run(
-        'CREATE CONSTRAINT ON (org:Organization) ASSERT org.name IS UNIQUE'
-      )
-    );
-    wait.push(
-      session.run(
-        'CREATE CONSTRAINT ON (org:Organization) ASSERT org.id IS UNIQUE'
-      )
-    );
-    wait.push(
-      session.run(
-        'CREATE CONSTRAINT ON (org:Organization) ASSERT exists(org.id)'
-      )
-    );
-    wait.push(
-      session.run(
-        'CREATE CONSTRAINT ON (org:Organization) ASSERT exists(org.name)'
-      )
-    );
-    wait.push(
-      session.run(
-        'CREATE CONSTRAINT ON (org:Organization) ASSERT exists(org.active)'
-      )
-    );
-
-    // LOCATION
-    wait.push(
-      session.run(
-        'CREATE CONSTRAINT ON (loc:Location) ASSERT loc.name IS UNIQUE'
-      )
-    );
-    wait.push(
-      session.run('CREATE CONSTRAINT ON (loc:Location) ASSERT loc.id IS UNIQUE')
-    );
-    wait.push(
-      session.run('CREATE CONSTRAINT ON (loc:Location) ASSERT exists(loc.id)')
-    );
-    wait.push(
-      session.run('CREATE CONSTRAINT ON (loc:Location) ASSERT exists(loc.name)')
-    );
-    wait.push(
-      session.run(
-        'CREATE CONSTRAINT ON (loc:Location) ASSERT exists(loc.active)'
-      )
-    );
-
-    // AREA
-    wait.push(
-      session.run('CREATE CONSTRAINT ON (area:Area) ASSERT area.name IS UNIQUE')
-    );
-    wait.push(
-      session.run('CREATE CONSTRAINT ON (area:Area) ASSERT area.id IS UNIQUE')
-    );
-    wait.push(
-      session.run('CREATE CONSTRAINT ON (area:Area) ASSERT exists(area.id)')
-    );
-    wait.push(
-      session.run('CREATE CONSTRAINT ON (area:Area) ASSERT exists(area.name)')
-    );
-    wait.push(
-      session.run('CREATE CONSTRAINT ON (area:Area) ASSERT exists(area.active)')
-    );
-
-    await Promise.all(wait);
-    session.close();
+export class AdminService implements OnApplicationBootstrap {
+  constructor(
+    private readonly db: DatabaseService,
+    private readonly config: ConfigService,
+    private readonly userService: UserService
+  ) {}
+
+  async onApplicationBootstrap(): Promise<void> {
+    if (!(await this.rootAdminSecurityGroupExists())) {
+      await this.createRootAdminSecurityGroup();
+    }
 
-    response.success = true;
+    if (!(await this.doesRootAdminUserAlreadyExist())) {
+      await this.createRootAdminUser();
+    }
 
-    return response;
+    await this.mergeRootAdminUserToSecurityGroup();
   }
 
-  async loadFakeData(): Promise<AdminOutputDto> {
-    const response = new AdminOutputDto();
-
-    const totalUsers = 50;
-    const totalOrgs = 50;
-
-    // ORGS
-    for (let i = 0; i < totalOrgs; i++) {
-      // this.orgService.create({
-      //   name: 'org_' + generate(),
-      // });
+  async rootAdminSecurityGroupExists(): Promise<boolean> {
+    const result = await this.db
+      .query()
+      .match([[node('sg', 'RootSecurityGroup')]])
```

Comment:
```suggestion
      .match(node('sg', 'RootSecurityGroup'))
```
Might be easier to read